python/extractSerpent2GCs.py

```diff
@@ -29,7 +32,7 @@ def makePropertiesDir(
         os.mkdir(outdir)
 
     # the constants moltres looks for:
-    goodStuff = ['BETA_EFF', 'Chit', 'Chid', 'lambda', 'Diffcoef', 'Kappa',
+    goodStuff = ['BETA_EFF', 'Chip', 'Chid', 'lambda', 'Diffcoef', 'Kappa',
```

Comment:
this is problematic.

Reply:
In the latest changes, I've added a boolean flag such that this script extracts ChiP and ChiD by default. Otherwise, it extracts ChiT.